ui/packages/consul-ui/tests/steps/interactions/form.js

```diff
@@ -11,7 +11,7 @@ export default function(scenario, find, fillIn, triggerKeyEvent, currentPage) {
       return page;
     } else {
       const $el = document.querySelector(`[name="${name}"]`);
-      fillIn($el, value);
+      await fillIn($el, value);
       return page;
```

Comment:
Thanks for adding the async/await here 👏 . I took a little more of a look into this, and it looks like the ember-cli-page-object we were previously using for this was not an async API.

I much prefer not using ember-cli-page-object to do this, but I _am_ curious as to why filling a text field is has an async API 🤔 , but I am glad we had failing tests due to the upgrade change 🎉